src/components/ec-full-screen-overlay/ec-full-screen-overlay.story.js

```diff
@@ -0,0 +1,24 @@
+import EcFullScreenOverlay from './ec-full-screen-overlay.vue';
+
+export default {
+  title: 'Full Screen Overlay',
+  component: EcFullScreenOverlay,
+};
+
+const Template = ({ modelValue, ...args }) => ({
+  components: { EcFullScreenOverlay },
+  setup() {
+    return {
+      args,
+    };
+  },
+  template: `
+    <ec-full-screen-overlay v-bind="args"></ec-full-screen-overlay>
+  `,
+});
+
+export const basic = Template.bind({});
+basic.args = {
+  title: 'I am a title',
+};
```

Comment:
Could you also add a story which displays more content in the component? Lots of parahraphs with Lorem ipsum texts is enough, but I'd like to see the overflow on this component working.